ci-operator/config/openshift-priv/hypershift/openshift-priv-hypershift-release-4.18.yaml

```diff
@@ -201,15 +201,13 @@ tests:
     cluster_profile: hypershift
     env:
       CLUSTER_TYPE_OVERRIDE: openstack-nfv
-      HYPERSHIFT_ZONES: us-east-1a,us-east-1b,us-east-1c
       NFV_NODEPOOLS: "true"
+      RHCOS_IMAGE_NAME: rhcos-4.18-hcp-nodepool
```

Comment:
Is this related or a latent bug? I don't see it mentioned in the commit message so I'm not really sure what's going on.

Reply:
Sorry I'll explain in the commit message.
It was a miss when I initially created the job. We need to override the image to be 4.18 otherwise we get the latest (4.19 now).